### PR TITLE
Feature / DAO-325 

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- Added new mobile menu proposal

### CODE REVIEW

File: packages/govern-console/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
 
-const NavBar = styled.nav`
+const NavBar = styled.nav<{ flexDirection: string }>`
+  padding: 16px;
   display: flex;
-  flex-direction: row:
-  gap: ${2 * GU}px;
-  padding: 8px 8px 18px 8px;
   align-items: center;
+  // background: rgba(255, 255, 255, 0.88);
```
Comment (githubdoramon) [inline packages/govern-console/src/components/Header/Header.tsx]: remove commented stuff

File: packages/govern-console/src/components/Header/Header.tsx [hunk 2]
```diff
@@ -55,61 +144,76 @@ const Header = () => {
...
-          display={layoutName === 'small' ? 'icon' : 'label'}
-          disabled={status === 'connecting'}
-        />
-      </RigtSideContainer>
+      </Container>
+      <Menu show={showMenu} isSmall={layoutIsSmall}>
+        <MenuItems isSmall={layoutIsSmall}>
+          {/* Currently unavailable
+          <NavLinkGroup gap={GU} isSmall={layoutIsSmall}>
+            <MenuLink to="/">Explore DAOs</MenuLink>
```
Comment (githubdoramon) [inline packages/govern-console/src/components/Header/Header.tsx]: Why the links are not available exactly?

File: packages/govern-console/src/components/Wallet/Wallet.tsx [hunk 3]
```diff
@@ -2,18 +2,21 @@ import { useState } from 'react';
...
 import { trackEvent, EventType } from 'services/analytics';
 import { useCallback } from 'react';
 
 //TODO add the icon for logged in users
 declare let window: any;
 
-const Wallet = ({}) => {
-  const connetButtonLabel = 'Connect wallet';
+type WalletProps = {
+  buttonCSS?: string;
```
Comment (githubdoramon) [inline packages/govern-console/src/components/Wallet/Wallet.tsx]: Why do we need external css for this button?

File: packages/govern-console/src/components/Header/Header.tsx (hunk 2 above)
Comment (Fabricevladimir) [reply packages/govern-console/src/components/Header/Header.tsx]: These links are on the proposal for the new header, which we aren't implementing yet.

Comment (githubdoramon) [reply packages/govern-console/src/components/Header/Header.tsx]: But we can use them and link to "/" for explore and to https://help.aragon.org/category/69-aragon-govern for Documentation

File: packages/govern-console/src/components/Wallet/Wallet.tsx (hunk 3 above)
Comment (Fabricevladimir) [reply packages/govern-console/src/components/Wallet/Wallet.tsx]: I seemed to have forgotten to pass it down!

**Review by githubdoramon — APPROVED** [review]

**Pull request merged**